pkg/kademlia/routing.go

```diff
@@ -44,8 +46,9 @@ type RoutingTable struct {
 }
 
 // NewRoutingTable returns a newly configured instance of a RoutingTable
-func NewRoutingTable(localNode pb.Node, kdb, ndb storage.KeyValueStore) (*RoutingTable, error) {
+func NewRoutingTable(logger *zap.Logger, localNode pb.Node, kdb, ndb storage.KeyValueStore) (*RoutingTable, error) {
```

Comment:
Adding here, but when specifying the logger argument, remember to properly prefix them with `outerLogger.Named("routing")`.